src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.controller;
+
+import jakarta.validation.Valid;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final DtoResponseMapper<UserResponseDto, User> responseMapper;
+
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    DtoResponseMapper<UserResponseDto, User> responseMapper) {
+        this.authenticationService = authenticationService;
+        this.responseMapper = responseMapper;
+    }
+
+    @PostMapping("/register")
+    UserResponseDto registration(@RequestBody @Valid UserRequestDto userRequestDto) {

```

Comment:
Don't forget to use access modifiers. Also, should we return anything in the registration method?